### ISSUE TITLE
TNS API Key naming between tom_setup app and tom_alerts is not consistent

### ISSUE THREAD

**Issue by blancmatter** [TOMToolkit/tom_base#363]
Checked in TOM Toolkit latest release version **v1.12.1**
The TNSBroker class gets the TNS API key using, `settings.BROKER_CREDENTIALS['TNS_APIKEY']`

However when setting up a new TOM the settings template  tom_setup/templates/tom_setup/settings.tmpl 
nests the info one lower down, e.g.

```python
ALERT_CREDENTIALS = {
    'TNS': {
        'api_key': ' '
    }
}
```

When this is populated it therefore fails to obtain the API key.

A quick fix was for me to change the settings.py file to provide this info in the format that the TNSBroker is expecting, e.g. ;

```python
BROKER_CREDENTIALS = {
    'TNS_APIKEY': ' '
}
```

I'm not sure if you wanted to change the settings.tmpl or TNSBroker code so didn't offer a pull request.

**Comment by dmcollom** [comment]
Thanks for pointing this out, Doug--I actually noticed it during a current feature I'm working on, and I'm planning to unify all of the brokers credentials under a `BROKERS` settings, which will match our existing `FACILITIES` settings in structure.

**Comment by dmcollom** [comment]
This was fixed in release 2.0.0. The new setting is `BROKERS`.
